### DISCUSSION TITLE
Content Management system for website

### DISCUSSION THREAD

**Discussion by ghost** [community/community#42599] [Other Feature Feedback, Questions, & Ideas]
Hi All, 

I'm a non technical founder at Siphter.io. We have an amazing website built on Laravel, JS, HTML, CSS etc. Currently, if I have to change any texts or typos on the website, I have to ask a developer to do it. Is there a plug and play solution available in the market that enables me to easily make text changes, add blogs etc myself? Without the need of a developer?

Thank you for your help!

Merry Christmas!

Dennis

**Comment by getlost01** [comment]
Yes, there are several options available that can allow you to make text changes and add blogs to your website without the need for a developer. One option is to use a content management system (CMS). A CMS is a web-based platform that allows you to create, edit, and publish content on your website without the need for technical skills.

Some popular CMS options include:

- WordPress: This is a widely used open-source CMS that offers a range of features and a user-friendly interface. It allows you to easily add and edit text and images on your website, as well as to create and publish new pages and blog posts.
- Joomla: This is another popular open-source CMS that offers a range of features and a user-friendly interface. It allows you to manage your website's content, including text and images, and to create and publish new pages and blog posts.
- Drupal: This is another open-source CMS that offers a range of features and a user-friendly interface. It allows you to manage your website's content, including text and images, and to create and publish new pages and blog posts.
- October CMS: This is an open-source CMS built on top of the Laravel framework. It offers a simple and intuitive interface that allows you to manage your website's content, including text and images, and to create and publish new pages and blog posts.
- PyroCMS: This is another open-source CMS built on Laravel. It offers a range of features, including a drag-and-drop page builder, a powerful editor, and support for multiple languages, which can make it easier for you to manage your website's content.
- Laravel Nova: This is a premium CMS built specifically for the Laravel framework. It offers a range of features, including a customizable dashboard, a powerful resource management system, and an intuitive interface that makes it easy to manage your website's content.

Another option to consider is using a website builder platform, such as Wix or Squarespace. These platforms allow you to create and customize a website without the need for technical skills, and offer a range of features and templates that can make it easy for you to add and edit text and images on your website.

**Reply by ghost** [reply]
Hi Getlost01, thank you very much for your comprehensive answer. Very helpfull.

**Reply by ghost** [reply]
Because we have already built the complete website and Laravel is our framework, I guess three options remain. 

- October CMS: This is an open-source CMS built on top of the Laravel framework. It offers a simple and intuitive interface that allows you to manage your website's content, including text and images, and to create and publish new pages and blog posts.
- PyroCMS: This is another open-source CMS built on Laravel. It offers a range of features, including a drag-and-drop page builder, a powerful editor, and support for multiple languages, which can make it easier for you to manage your website's content.
- Laravel Nova: This is a premium CMS built specifically for the Laravel framework. It offers a range of features, including a customizable dashboard, a powerful resource management system, and an intuitive interface that makes it easy to manage your website's content.


I will check them out to get an understanding which one would be the best/ smartest choice.

Thank you very much for your help!

Warm regards,
Dennis

**Reply by ghost** [reply]
How much time does it take for a senior dev to implementPyroCMS or Laravel Nova? Is that days, weeks?

**Reply by jfullstackdev** [reply]
that will depend on how complex is the system, it's actually the hardest part in planning: to determine how much time is needed for a certain sprint / module

**Comment by jfullstackdev** [comment]
if it's UI, you can simply learn HTML, CSS and it's only at the view of Laravel project

you may want to try that, that's the easiest task that can ever be given to a developer and you don't need to learn the backend or routing, just the view components of the Laravel

who knows, your interest might grow while you are learning that

**Reply by ghost** [reply]
Yes that's an option too, but for the future it's better to structure it good immediately. We will have marketing people working om this in 2023 I hope.
